src/Status.js

```diff
@@ -0,0 +1,31 @@
+const faces = new Set(["JACK", "QUEEN", "KING"]);
+
+export function Status({ cards }) {
```

Comment:
I know this is not the standard at Codecademy but can these components all export default?

Reply:
Haha sure!